### PR TITLE
amend ironsSource data

### PR DESCRIPTION
**Pull request by ghost**
- Amend company website
- Add ironSource owned company to ironSource
- Add Soomla tracker

[soom.la](soom.la) redirects to [https://www.is.com/AdQuality](https://www.is.com/AdQuality).

### CODE REVIEW

File: source/trackers.json [hunk 1]
```diff
@@ -1200,6 +1212,7 @@
         "iq.com": "iqiyi",
         "ironsrc.com": "ironsource",
         "ironsrc.net": "ironsource",
+        "soom.la": "ironsource",
```
Comment (jellizaveta) [inline source/trackers.json:1215]: Is the domain `soom.la` from `ironsource` tracker or from `soomla` ?

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: source/trackers.json (hunk 1 above)
Comment (Alex-302) [reply source/trackers.json:1215]: image

Comment (Alex-302) [reply source/trackers.json:1215]: not sure if we need their old domains, which redirect to is.com

Comment (Alex-302) [reply source/trackers.json:1215]: In general this is a regular site. We can't add all existing sites to our base.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**